Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -7004,7 +7006,7 @@ static NSTextAlignment nsTextAlignmentFromTextAlignment(TextAlignment textAlignm
         [m_topScrollPocket layout];
         captureView = [m_topScrollPocket captureView];
         [m_layerHostingView addSubview:captureView.get() positioned:NSWindowBelow relativeTo:nil];
-        [captureView layer].zPosition = -CGFLOAT_MAX;
+        [captureView layer].zPosition = -FLT_MAX;
```

Comment:
I'm more of a `std::numeric_limits<float>::lowest()` person myself 😛

Reply:
Works for me!